java.source.base/src/org/netbeans/api/java/source/TreeUtilities.java

```diff
@@ -1883,6 +1883,51 @@ public boolean isEndOfCompoundVariableDeclaration(@NonNull Tree tree) {
         return false;
     }
  
+    /**
+     * Checks whether tree is part of compound variable declaration.
+     * {@link Tree}.
+     *
+     * @param tree the tree{@link Tree}
+     * @return true if tree is part of compound variable declaration else return
+     * false.
```

Comment:
add `@since` tag. Fix the `@link`, this IMHO displays as treeTree. Also putting just link to Tree as a sole description of the operation is strange. If you want to emphasize relationship to Tree type, use `@link` javadoc tag (same as `@return`). Maybe it could be sufficient to link the type from the parameter description.
Possibly use `{@code true} {@code false}` for well-known constants, they'll print in monospaced font.